packages/notifi-dapp-example/src/components/WalletSelectModal.tsx

```diff
@@ -32,34 +32,35 @@ export const WalletSelectModal: FC<WalletSelectModalProps> = ({
         onClick={() => setIsOpenWalletsModal(false)}
       ></div>
       <div
-        className={`flex flex-col fixed w-90 min-h-72 md:min-w-[40rem] md:h-80 border border-notifi-card-border bg-notifi-destination-card-bg rounded-xl ${
+        className={`flex flex-col fixed w-90 min-h-72 md:min-w-[681px] md:h-[452px] border border-notifi-card-border bg-notifi-destination-card-bg rounded-xl ${
           Object.values(allWallets)
             .map((wallet) => wallet.isInstalled)
             .includes(true)
             ? ''
             : 'hidden'
         } sm:flex`}
       >
-        <div className=" flex h-14 justify-center items-end">
+        <div className="relative flex justify-center items-end mt-12">
           <div className="font-medium text-xl text-notifi-text">
             Select your wallet
           </div>
+          <Icon
+            id="close-icon"
+            className="absolute right-8 text-gray-500 cursor-pointer"
+            onClick={() => setIsOpenWalletsModal(false)}
+          />
         </div>
-        <Icon
-          id="close-icon"
-          className="absolute right-8 top-8 text-gray-500 cursor-pointer"
-          onClick={() => setIsOpenWalletsModal(false)}
-        />
-        <div className="flex grow gap-6 p-5 justify-center items-center flex-wrap">
+
+        <div className="flex gap-6 px-12 pt-10 justify-center items-center flex-wrap">
           {objectKeys(wallets)
             .filter(
-              (wallet) => wallets[wallet].isInstalled && wallet === 'metamask',
+              (wallet) => wallets[wallet].isInstalled && wallet !== 'keplr',
```

Comment:
Is there a better way to sort out the wallets that we need for GMX instead of use wallet !== 'keplr'? as we may add more wallets in wallet provide in the future.